yb-voyager/cmd/migration_complexity.go

```diff
@@ -200,3 +210,121 @@ func getComplexityForLevel(level string, count int) string {
 		panic(fmt.Sprintf("unknown impact level %s for determining complexity", level))
 	}
 }
+
+// ======================================= Migration Complexity Explanation ==========================================
+
+const explainTemplateHTML = `
+<table border="1" cellpadding="5" cellspacing="0" style="border-collapse: collapse;">
+    <thead>
+        <tr>
+            <th>Category</th>
+            <th>Level-1</th>
+            <th>Level-2</th>
+            <th>Level-3</th>
+            <th>Total</th>
+        </tr>
+    </thead>
+    <tbody>
+    {{- range .Summaries }}
+        <tr>
+            <td>{{ .Category }}</td>
+            <td>{{ index .ImpactCounts "LEVEL_1" }}</td>
+            <td>{{ index .ImpactCounts "LEVEL_2" }}</td>
+            <td>{{ index .ImpactCounts "LEVEL_3" }}</td>
+            <td>{{ .TotalIssueCount }}</td>
+        </tr>
+    {{- end }}
+    </tbody>
+</table>
+
+<p>
+	Level-1 Impact: Resolutions are available with minimal effort.<br/>
+	Level-2 Impact: Resolutions are available requiring moderate effort.<br/>
+	Level-3 Impact: Resolutions may not be available or are highly complex.
```

Comment:
Resolutions may not be available or are highly complex -> Resolutions may not be available or are complex ->

Reply:
yes, thanks.